search-enterprise/latest/config/transactional.md

```diff
@@ -0,0 +1,382 @@
+---
+title: Transactional metadata query
+---
+
+Alfresco Content Services supports the execution of a subset of the CMIS Query Language (CMIS QL) and Alfresco Full Text Search (AFTS) queries directly against the database. Also, the noindex subsystem supports queries only against the database. This collection of features is called transactional metadata query (TMDQ).
+
+TMDQ supports use cases where eventual consistency is not the preferred option.
+
+The Elasticsearch subsystem is eventually consistent. A change can take any length of time to be reflected in the index, ranging from a few seconds to several minutes. Elasticsearch indexes the metadata and the content of each updated node, in the order in which the nodes were last changed. The indexing components will try to index information about nodes as fast as possible, but content indexing is likely to be limited by the time needed to extract text from the files and all indexing will be affected by the rate at which the nodes are being changed.
```

Comment:
We have made effort to improve eventual consistency with Elasticsearch, and I think we should update this para to reflect this. Suggest: 
Replace:
"A change can take any length of time to be reflected in the index, ranging from a few seconds to several minutes."
With
"The amount of time a change a change takes to reflect in the index is normally less than 1 second, but can be longer under heavy load, or for complex/cascading updates."

Reply:
Re checked wasnt there. Sorry.